src/models/flinkStatement.ts

```diff
@@ -167,19 +201,18 @@ export class FlinkStatement implements IResourceBase, IdItem, ISearchable, IEnvP
 
   /**
    * For statement results to be viewable, it must satisfy these conditions:
-   * 1. The statement must NOT be a background statement (an INSERT INTO statement)
-   * 2. The statement must have been created in the last 24 hours
-   *    (which is the TTL for the statement result to be deleted)
-   * 3. The statement phase must be either RUNNING or COMPLETED.
+   * 1. The statement must have been created in the last 24 hours
+   *    (which is the TTL for the statement result to be deleted.)
+   * 2. The statement phase indicates viewable {@link VIEWABLE_PHASES viewable.}
```

Comment:
Minor fix since it shows a not-linked `viewable` with a linked `viewable`
```suggestion
   * 2. The statement phase indicates {@link VIEWABLE_PHASES viewable.}
```

Reply:
Grr, duh

Reply:
Now:

<img width="927" alt="image" src="https://github.com/user-attachments/assets/3cd443be-1961-44b2-8411-4af3c269ade5" />